tract_querier/tract_math/decorator.py

```diff
@@ -67,10 +104,13 @@ def wrapper(*args):
             ):
                 if args[-1] == '-':
                     args = args[:-1]
-                process_output(func(*args))
+
```

Comment:
Maybe here we could add something like

```python
has_optional_arguments = 'optional_arguments' in argspec.args
if has_optional_arguments:
   option_and_args=(options_dict,)+args
```
such that if a function doesn't have optional arguments it's not forced too